flink-runtime-web/web-dashboard/src/app/components/humanize-duration.pipe.ts

```diff
@@ -46,23 +46,23 @@ export class HumanizeDurationPipe implements PipeTransform {
             if (seconds === 0) {
               return `${ms}ms`;
             } else {
-              return `${seconds}s`;
+              return `${seconds}s ${ms}ms`;
             }
           } else {
-            return `${minutes}m ${seconds}s`;
+            return `${minutes}m ${seconds}s ${ms}ms`;
           }
         } else {
           if (short) {
             return `${hours}h ${minutes}m`;
           } else {
-            return `${hours}h ${minutes}m ${seconds}s`;
+            return `${hours}h ${minutes}m ${seconds}s ${ms}ms`;
           }
         }
       } else {
         if (short) {
           return `${days}d ${hours}h`;
         } else {
-          return `${days}d ${hours}h ${minutes}m ${seconds}s`;
+          return `${days}d ${hours}h ${minutes}m ${seconds}s ${ms}ms`;
```

Comment:
nit: I think the explicit ms might be useless for jobs longer than 1h.

Reply:
Thanks for your suggestion, it has been fixed.